src/pages/edit_resource/access_and_licence_step/LicenseWizardFields.tsx

```diff
@@ -125,29 +139,81 @@ const LicenseWizardFields: FC<LicenseWizardFieldsProps> = ({
 
   const handleChangeInExtraRestriction = async (event: React.ChangeEvent<HTMLInputElement>) => {
     handleChange(event);
+    setFieldValue(ResourceFeatureNamesFullPath.ResourceRestriction, event.target.value);
+    try {
+      setAllChangesSaved(false);
+      setSavingResourceRestrictionError(undefined);
+      const promiseArray: Promise<any>[] = [];
+      promiseArray.push(
+        postResourceFeature(values.identifier, ResourceFeatureNames.ResourceRestriction, event.target.value)
+      );
+      if (event.target.value !== LicenseRestrictionOptions.yes) {
+        promiseArray.push(postResourceFeature(values.identifier, ResourceFeatureNames.CanBeUsedCommercially, ''));
+        promiseArray.push(postResourceFeature(values.identifier, ResourceFeatureNames.OthersCanModifyAndBuildUpon, ''));
+      }
+      await Promise.all(promiseArray);
+      setAllChangesSaved(true);
+      resetFormButKeepTouched(touched, resetForm, values, setTouched);
+    } catch (error) {
+      setSavingResourceRestrictionError(error);
+    }
     await calculatePreferredLicense(
```

Comment:
litt usikker på om jeg burde lagt denne inni try/catch-blokka også, men vi må jo ikke vente på den for å få endret radio-knappene

Reply:
Tenker den enten burde vært inni try catch blokka, eller at calculatePrefferedLicense får sin egen try-catch blokk. Heller mest mot det første.